### PR TITLE
feat(discv2): event breakdown

### PR DESCRIPTION
**Pull request by doralchan**
Screen Shot 2019-12-18 at 12 12 23 PM

### CODE REVIEW

File: src/sentry/static/sentry/app/views/eventsV2/eventDetails/transaction/eventBreakdown.tsx [hunk 1]
```diff
@@ -157,11 +162,68 @@ class EventBreakdown extends React.Component<Props> {
...
-    // TODO: Dora to take over
-    // const results = this.generateCounts();
-
-    return null;
+    const results = this.generateStats();
+
+    return (
+      <StyledBreakdown>
+        <SectionHeading>{t('Ops Breakdown')}</SectionHeading>
+        {results.ops.map(currentOp => {
```
Comment (leedongwei) [inline src/sentry/static/sentry/app/views/eventsV2/eventDetails/transaction/eventBreakdown.tsx]: Nit: You can shorten this to `currOp`. I like to use `prev`, `curr`, `next` to prefix variables because they are all 4 chars.

**Review by leedongwei — APPROVED** [review]

File: src/sentry/static/sentry/app/views/eventsV2/eventDetails/transaction/eventBreakdown.tsx (hunk 1 above)
Comment (doralchan) [reply src/sentry/static/sentry/app/views/eventsV2/eventDetails/transaction/eventBreakdown.tsx]: Will do

**Review by dashed — APPROVED** [review]
Looks good to me 👍
